### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by jacob6838**
Creating initial CI CD GCP function deployment pipeline

### CODE REVIEW

**Comment by jacob6838** [comment]
This is a test very simple pipeline

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#   displayName: 'deploy icone function'
+
+# - script:
+#     gcloud auth activate-service-account --key-file $(authkey.secureFilePath)
+#     gcloud functions deploy salesforce-data --runtime--region=us-central1 --project=cdot-oim-wzdx-prod
+#   displayName: 'deploy cotrip function'
+
+- script:
+    # gcloud auth activate-service-account --key-file $(authkey.secureFilePath)
+    gcloud functions deploy navjoy-568-translator--region=us-central1 --project=cdot-oim-wzdx-dev
```
Comment (payneBrandon) [inline azure-pipelines.yml]: Do we already have a build for `navjoy-568-translator` that this just pulls from? Also we probably need a space between `navjoy-568-translator` and the `--region` parameter.

Comment (jacob6838) [reply azure-pipelines.yml]: Good point on the space, just fixed that. It is configured to build in GCP, so it will just deploy the whole local folder

**Review by payneBrandon — APPROVED** [review]
Looks good to me. Once we have this POC working, we'll want to abstract out the project name so we can use the same pipeline across projects.

**Pull request merged**